### ISSUE TITLE
Lineage OS can't be found

### ISSUE THREAD

**Issue by StefanoD** [MariusQuabeck/magic-device-tool#103]
I have Fedora 26, One Plus One and installed magic-device-tool via snap.
I switched to the root user via `su` and launched the tool.
After unlocking the device, I wanted to install LineageOS with Google Apps. But at some point, I get this error:

>  → LineageOS 14.1 zip 
> cannot stat '/root/snap/magic-device-tool/255/.cache/magic-device-tool/lineage-14.1-20170724-nightly-bacon-signed.zip': No such file or directory

**Comment by MariusQuabeck** [comment]
Thanks I'll look into this

**Comment by MariusQuabeck** [comment]
Hey @StefanoD,

I've tried it a few times and it worked fine. Usually, I would suggest trying a different cable but it makes no sense as it's saying that it can't find the file. I have no idea why this is happening to you, I've tried the snap on Ubuntu and Solus and it worked fine. I don't know if there is any difference in the snapd implementation on fedora. I'll add `su` / `sudo bash` to the instructions tho

**Comment by StefanoD** [comment]
I looked into this and it seems, it fails on the first adb command.

**Comment by MariusQuabeck** [comment]
Yeah thats the adb push part but this also worked fine here. I'll try on fedora when I'm back home from holiday and get back to you 

![bildschirmfoto vom 2017-08-19 13-05-44](https://user-images.githubusercontent.com/17063564/29486178-73c02c9c-84e0-11e7-84e6-30ec4bb180e1.png)

**Comment by StefanoD** [comment]
I installed Ubuntu 17.10 Alpha/Beta and it worked. :)

**Comment by benfeatherstone** [comment]
I have the same issue on Ubuntu Gnome 16.04.3. It downloaded it fine the first time, but then couldn't detect device. On the second attempt, it couldn't find the file.

> You may see a prompt asking you for read/write permissions
Ignore that prompt, the tool will take care of the installation

>  → LineageOS 14.1 zip 
cannot stat '/home/ben/.cache/magic-device-tool/lineage-14.1-20170530-nightly-hammerhead-signed.zip': No such file or directory

**Comment by MariusQuabeck** [comment]
@benfeatherstone please create a new issue for this. your problem happens on a different distro with a different device. or don't. I can't really do anything about the snapd implementation on different distros :(

**Comment by stuiterveer** [comment]
@MariusQuabeck You probably could not reproduce the issue since you already have the necessary firmware downloaded from earlier. For new installations of MDT this issue is easily reproducable. Please try it with a new install or with the cache folder empty. The issue is caused due to the fact that it tries downloading a file that's hardcoded but not available anymore (see #106). The best way to go about this without having to update the install script for each new version of Lineage OS is to make sure the tool automatically fetches the URL for the latest version.
